RachaelNantale/property-pro-lite
# set up node project

#### What does this PR do?
- This PR sets up the node project together with the linting
#### Description of Task to be completed?
- Set up of the node/express project
- Set up linting library
#### How should this be manually tested?
- run `npm run dev` to run the app
- run `npm run lint` to see whether there are linting errors
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
